src/config/karma.conf.js

```diff
@@ -21,9 +21,6 @@ const env = {
 // Webpack overrides for karma
 const karmaWebpackConfig = merge.strategy({ plugins: 'replace' })(webpackConfig(), {
   entry: '',
-  output: {
-    libraryTarget: 'var'
```

Comment:
I'm not sure why this is/was necessary, but the added test passes with `globalObject` as `'self'` or `'window'` unless this is removed, in which case it passes with `globalObject` as `'self'` and fails with it as `'window'` as reported in ipfs/js-ipfs#2349.

The js-IPFS tests pass without this block when run in the browser/webworkers so I'm not sure if it's necessary - https://github.com/ipfs/js-ipfs/pull/3343